GVRf/Framework/framework/src/main/jni/vulkan/vulkan_shader.cpp

```diff
@@ -113,8 +113,11 @@ int VulkanShader::bindTextures(VulkanMaterial* material, std::vector<VkWriteDesc
         const DataDescriptor::DataEntry* e = mTextureDesc.find(texname);
         if ((e == NULL) || e->NotUsed)
         {
-            return;
+            return -1;
```

Comment:
There should be no return value. This is what func is: ``std::function< void(const char* texname, Texture* tex) >``. It actually crashes if compiled with clang.